web/src/index.html

```diff
@@ -47,7 +47,7 @@ <h1 class="title">Game</h1>
           </div>
           <div class="game">
           <!-- Test game Canvas -->
-            <canvas id="gameCanvas" width="900" height="600" style="border:1px solid #000000; background-color:black;"></canvas>
+            <canvas id="gameCanvas" width="900" height="600"></canvas>
```

Comment:
we will talk about that when we determine the actual size of the map and stuff, right now the size values are only temporary, I'll make an issue about it.